config.py

```diff
@@ -1,5 +1,10 @@
 
 import os
 
-chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/chromedriver.exe'
+windows_chromedriver = 'chromedriver.exe'
```

Comment:
Revert this change. My understanding is that if you change the chromedriver filename it will cause errors, and since mac and linux chromedriver files are identically named we're unlikely to implement multi-OS support this way.